### ISSUE TITLE
[1.20.1_1.7.0] Foxes and Ocelots aren't working

### ISSUE THREAD

**Issue by Flexico** [AlexModGuy/DomesticationInnovation#152]
Foxes sit and stay when told to wander, and Ocelots don't respond to right-clicking at all!

**Comment by Daralexen05** [comment]
Ocelots can be tamed?

**Comment by Flexico** [comment]
Yeah, kind of! You can gain their trust the same way as you can with foxes, but even when you do they still despawn if you don't name them. It seems like a partially-done mechanic
